connections/index.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Monitoring Postgres connections'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+The Connections page in pganalyze tells you about the database connections,
+using the data from the `pg_stat_activity`.
+Select the server to get the overall connections info, or select the database
+as well to get more detailed data such as wait events per database.
+
+You can check the history of connection states in the History chart. This chart
+is using the data collected every 10 minutes. This section helps you to
+understand the trend of the total count of connections or some unusual pattern
+of the connections such as the increase of the Waiting For Lock connections.
+
+## Connection States tab
+
+![](connection-states.png)
+
+The Connection States tab has the History chart and the Connection Traces
+panel. The Connection Traces panel shows what kind of queries are running
+currently, with the automatic refresh (will be paused after a while).
+You can also jump to the certain time point's Connection Traces by clicking
+the specific time of the History chart.
+
+The Connection Traces panel is only available on the [Scale Plan](/pricing)
+and higher.
+
+### Lock Monitoring
+
+You can monitor various wait events including locks with the Wait Events tab.
+The History chart of the Connection States tab also gives you the historical
+count of the Waiting For Lock state connections.
+
+With the Lock Monitoring, you can dig into this lock information deeper. Within
+Connection Traces panel in Connection States tab, whenever the connection is
+waiting for a lock, the background color of the row will be highlighted. For
+the connection that is blocking other queries is also highlighted.
```

Comment:
```suggestion
Connections that are blocking other queries are also highlighted.
```
There could be multiple separate blocking connections, right?

Reply:
yup